org.archivekeep.ArchiveKeep.appdata.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>org.archivekeep.ArchiveKeep</id>
+    <launchable type="desktop-id">org.archivekeep.ArchiveKeep.desktop</launchable>
+
+    <name>ArchiveKeep</name>
+    <summary>Preserve files in multiple storages</summary>
+    <description>
+        <p>
+            <em>DISCLAIMER:</em>This is an early release. The author already uses the software to fulfil own needs, but there's still a room for improvement and a potential to find undiscovered issues, bugs or edge-cases.
+        </p>
+        <p>Utility facilitating preservation of personal archive of files, replicated in multiple storages of own choice.</p>
+        <p>The main focus is on preservation - keep identical copies while not performing any destructive operations, nor replication of corruption, nor other means of data loss.</p>
+        <p>Features:</p>
+        <ul>
+            <li>Plain files - archived files aren't wrapped in custom format, and are readable with any application.</li>
+            <li>Incremental synchronization - update offline and other storages incrementally, when they become available.</li>
+            <li>Filesystem agnostic</li>
+        </ul>
+    </description>
+
+    <screenshots>
+        <screenshot type="default">
+            <caption>Main window</caption>
+            <image>https://archivekeep.org/generated_screenshots/main-window.png</image>
+        </screenshot>
+    </screenshots>
+
+    <url type="homepage">https://archivekeep.org/</url>
+
+    <provides>
+        <binary>archivekeep-desktop</binary>
+    </provides>
+
+    <metadata_license>FSFAP</metadata_license>
+    <project_license>AGPL-3.0-or-later</project_license>
+
+    <developer id="org.kravemir">
+        <name>Miroslav Kravec</name>
+    </developer>
+
+    <content_rating type="oars-1.0" />
```

Comment:
```suggestion
    <content_rating type="oars-1.1" />
```

Reply:
:heavy_check_mark: - applied to upstreamed appstream metainfo